api/v1/utils/contests.js

```diff
@@ -1,22 +1,11 @@
-const { validationResult } = require("express-validator");
-
-const { ERRORS } = require("../../constants");
-const { LogicError } = require("../../utils/error-classes");
-
-function dateToTimestamp(date) {
-  return date.getTime() / 1000;
-}
-
-function timestampToDate(timestamp) {
-  return new Date(timestamp * 1000);
-}
+const utils = require("./index");
```

Comment:
`index.js` should import other files, not the other way.

Move shared utils to a file named `common.js` or `shared.js` then import it like:
```
const commonUtils = require("./common");
```